### DISCUSSION TITLE
IK_path is not completed, the planning failed and the code stop

### DISCUSSION THREAD

**Discussion by Rabibabo** [Victorlouisdg/cloth-competition#22] [Questions] [answered]
the information just like this:
![2024-05-02 10-31-33 的屏幕截图](https://github.com/Victorlouisdg/cloth-competition/assets/130025759/08ca6732-c772-4b28-8aff-5913652dee44)
![2024-05-02 10-32-26 的屏幕截图](https://github.com/Victorlouisdg/cloth-competition/assets/130025759/529ac643-6643-40b9-88e4-3e7496b69770)

Have you encountered this kind of problem before? It seems not a error brought by given grasp_pose. How can I sovle it?

**Comment by Victorlouisdg** [comment, marked as answer]
Yes, I believe this is a bug in an airo-drake function that I wrote.The fix has already been committed:
https://github.com/airo-ugent/airo-drake/commit/59a69607ee2831664a9082aed52324328ea2e99c

The function did not correctly handle the situation when the IK gave 0 joint solutions for a TCP pose. You can also get the fix by upgrading airo-drake to its latest version:
```
pip install airo-drake==0.0.5
```
